tests/hash-backend-integration/src/tests/subgraph/circular.test.ts

```diff
@@ -0,0 +1,17 @@
+import path from "node:path";
+
+import { resetGraph, restoreSnapshot } from "../test-server";
+
+jest.setTimeout(60000);
+
+afterAll(async () => {
+  await resetGraph();
+});
+
+describe("Circular Snapshot", () => {
+  it("can upload snapshot", async () => {
```

Comment:
Should we call this restore instead of upload?
```suggestion
  it("can restore snapshot", async () => {
```